plugins/common/opcua/opcua_util.go

```diff
@@ -157,6 +157,7 @@ func (o *OpcUAClient) generateClientOpts(endpoints []*ua.EndpointDescription) ([
 		opcua.ApplicationURI(appuri),
 		opcua.ApplicationName(appname),
 		opcua.RequestTimeout(time.Duration(o.Config.RequestTimeout)),
+		opcua.SessionTimeout(time.Duration(o.Config.SessionTimeout)),
```

Comment:
Shouldn't this only be set if the timeout is actually specified by the user? Otherwise you will override the default and change the behavior of current configs, don't you?

Reply:
done